Assets/PatchKit Patcher/Scripts/AppData/Local/LocalMetaData.cs

```diff
@@ -183,10 +204,14 @@ public string GetProductKey()
         
         public void SetMainExecutableAndArgs(string mainExecutable, string mainExecutableArgs)
         {
-            _data.MainExecutable = mainExecutable;
-            _data.MainExecutableArgs = mainExecutableArgs;
+            if (!_data.MainExecutable.Contains(mainExecutable) ||
+                !_data.MainExecutableArgs.Contains(mainExecutableArgs))
```

Comment:
Instead of checking for "Contains" simply compare two strings.

Reply:
Done